### PR TITLE
Release eds-core-react

### PR DESCRIPTION
**Pull request by wenche**
Part of #965 

### Added ✨

- `SingeSelect` component ([#896](https://github.com/equinor/design-system/issues/896))
- `MultiSelect` component ([#897](https://github.com/equinor/design-system/issues/897))

### Fixed 🐛

- Hover size on ghost buttons ([#595](https://github.com/equinor/design-system/issues/595))
- Fix wrong spacings inside `Chip` causing the content to not be proper centered ([#956](https://github.com/equinor/design-system/issues/956))

### Changed 📓

- Use base tokens in `Button` component ([#831](https://github.com/equinor/design-system/issues/831))
- Use base tokens in `Table` component ([#830](https://github.com/equinor/design-system/issues/830))
- Support for React 17 ([#813](https://github.com/equinor/design-system/issues/813))
- Update dependencies ([#904](https://github.com/equinor/design-system/issues/904))

### CODE REVIEW

**Comment by wenche** [comment]
Uhm, was the Table and Button base token stuffs in `eds-tokens` or in `eds-core-react` @mimarz

**Comment by mimarz** [comment]
> Uhm, was the Table and Button base token stuffs in `eds-tokens` or in `eds-core-react` @mimarz

Both. Updated types and such are under types in tokens and we also removed the `table` and `button` component tokens from `eds-tokens`. Changes to `eds-core-react` are well, changes to use the base tokens and token types :)

File: libraries/core-react/CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,25 @@ All notable changes to this project will be documented in this file.
...
+
+- Fix wrong spacings inside `Chip` causing the content to not be proper centered ([#956](https://github.com/equinor/design-system/issues/956))
+
+### Changed 📓
+
+- Use base tokens in `Button` component ([#831](https://github.com/equinor/design-system/issues/831))
+- Use base tokens in `Table` component ([#830](https://github.com/equinor/design-system/issues/830))
+- Support for React 17 ([#813](https://github.com/equinor/design-system/issues/813))
+- Update dependencies ([#904](https://github.com/equinor/design-system/issues/904))
+- Hover size on ghost buttons ([#595](https://github.com/equinor/design-system/issues/595))
```
Comment (mimarz) [inline libraries/core-react/CHANGELOG.md:25]: I wong say this is a bugfix, it was to small to begin with

**Review by mimarz — CHANGES_REQUESTED** [review]
Just a small change in log, else good

File: libraries/core-react/CHANGELOG.md (hunk 1 above)
Comment (wenche) [reply libraries/core-react/CHANGELOG.md:25]: OK, will remove the removal then 🥇

**Review by mimarz — APPROVED** [review]
LGTM 👍

**Pull request merged**